src/Atlas.cc

```diff
@@ -352,6 +354,17 @@ void Atlas::PostLoad()
     mvpBackupMaps.clear();
 }
 
+void Atlas::SaveAtlas(string pathSaveFileName, string strVocabularyName, string strVocabularyChecksum) {
```

Comment:
Whats the distinction between using SaveAtlas here and the SaveAtlas in System.cc?

Reply:
Mainly motivated by having to lock the `mMutexAtlas` which I couldn't do from inside `System.cc`